docs/SUMMARY.md

```diff
@@ -1,15 +1,15 @@
 # Table of contents
 
-- [Introduction](../README.md)
+- [Introduction](./readme.md)
 - [Airbyte Cloud QuickStart](cloud/getting-started-with-airbyte-cloud.md)
   - [Core Concepts](cloud/core-concepts.md)
   - [Managing Airbyte Cloud](cloud/managing-airbyte-cloud.md)
-- [Airbyte Open Source Quickstart](quickstart/README.md)
+- [Airbyte Open Source Quickstart](quickstart/deploy-airbyte.md)
```

Comment:
this readme and some others didn't exist.  Links to folders without a readme like `/quickstart/` don't work in docusaurus.  I pointed this and other top level links to a markdown file in the folder

Reply:
@git-phu I haven't looked at the documentation... though this conversation is probably about to change that.  From testing, docusaurus doesn't appear to understand `/folders/` I believe this is a quirk of the manual sidebar we are using.  As stated elsewhere I think I'll just make a quick README in the places that didn't have one